modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -49,6 +51,29 @@ exports.update = function (req, res) {
 };
 
 /**
+ * Update the user's classes
+ */
+ exports.updateclasses = function (req, res) {
+    var user = User.findById(req.userId);
+    var classes;
+
+    //console.log('classes');
+    classes.idnumber = req.classes._id;
+    classes.courseCode = req.classes.courseCode;
+    classes.name = req.classes.name;
+
```

Comment:
instead of creating the var class here, you can create the class object in client/controller so that server.contoller.js would be much cleaner. @tjsparks5  told me that this is not working, so I might fix it later if you want me to

Reply:
We are working on it. If you'd like to help that'd be great.